data/card_definitions.json

```diff
@@ -338,6 +338,458 @@
 	},
 
 
+	{
+		"NOTE": "ARAKUNE CARDS START",
+		"id": "arakune_ntoinfinity",
+		"type": "ultra",
+		"display_name": "n to infinity",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 4,
+		"power": 5,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "before",
+				"effect_type": "close",
+				"amount": 3
+			},
+			{
+				"timing": "after",
+				"condition": "opponent_stunned",
+				"effect_type": "take_damage",
+				"opponent": true,
+				"amount": 99
+			}
+
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Burst",
+			"effects": [
+				{
+					"timing": "immediate",
+					"effect_type": "draw",
+					"amount": 2
+				},
+				{
+					"timing": "immediate",
+					"effect_type": "seal_this"
+				},
+				{
+					"timing": "immediate",
+					"effect_type": "bonus_action"
+				}
+			]
+		}
+	},
+	{
+		"id": "arakune_finverse",
+		"type": "ultra",
+		"display_name": "f inverse",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 2,
+		"range_max": 2,
+		"speed": 7,
+		"power": 4,
+		"armor": 0,
+		"guard": 0,
+		"on_invalid": {
+			"effect_type": "armorup",
+			"amount": 3
+		},
+		"effects": [
+			{
+				"timing": "hit",
+				"condition": "range",
+				"condition_amount": 1,
+				"effect_type": "powerup",
+				"amount": 2
+			},
+			{
+				"timing": "after",
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "push", "amount": 1 },
+					{ "effect_type": "push", "amount": 2 },
+					{ "effect_type": "pull", "amount": 1 },
+					{ "effect_type": "pull", "amount": 2 }
+				]
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 1,
+			"cancel_cost": -1,
+			"display_name": "Forbidden Knowledge",
+			"effects": [
+				{
+					"timing": "immediate",
```

Comment:
Does this boost have the same potential ordering issues as Plague Knight's boost that looks at the top 3 cards of deck? honestly i have no idea how you'd reasonably get around it with 5 cards so I hope not but

Reply:
I think this is okay. and even if it isn't, too bad.